helix-rest/src/main/java/org/apache/helix/rest/metadatastore/accessor/ZkRoutingDataWriter.java

```diff
@@ -40,8 +52,11 @@
   private final String _namespace;
   private final HelixZkClient _zkClient;
   private final ZkDistributedLeaderElection _leaderElection;
+  private final CloseableHttpClient _forwardHttpClient;
+  private final String _myHostName;
 
   public ZkRoutingDataWriter(String namespace, String zkAddress) {
+    System.out.println("writer created namespace=" + namespace + " zkAddress=" + zkAddress);
```

Comment:
Remove print statements? We want to avoid this kind of stuff in production code.

Reply:
This one escaped my own review. Fixing.